TripKit/Classes/helpers/TKAccessibilityHelper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  TKAccessibilityHelper.swift
+//  TripKit-iOS
+//
+//  Created by Kuan Lun Huang on 22/2/18.
+//  Copyright © 2018 SkedGo. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+@objc public class TKAccessibilityHelper: NSObject {
+  
+  let region: SVKRegion
+  
+  @objc public var isWheelchairAvailable: Bool {
+    return rx_wheelchairAvailable.value
+  }
+  
+  private let rx_regionInfo: Variable<API.RegionInfo?> = Variable(nil)
+  private let rx_wheelchairAvailable = Variable(false)
+  public let rx_wheelchairAvailabilityUpdated = PublishSubject<Bool>()
```

Comment:
What does this mean? When would this update?

Reply:
No longer relevant